taxonomies/labels.txt

```diff
@@ -80,22 +83,28 @@ synonyms:fr:fabrique, fabriqué
 synonyms:ru:изготовление, производство
 synonyms:es:fabricación, producción, elaborado
 
+synonyms:en:fats, fat
 synonyms:fr:matières grasses, MG, gras
 synonyms:ru:жир
 
+synonyms:en:natural, of natural origin
 synonyms:fr:naturel, d'origine naturelle, origine naturelle
 synonyms:es:natural, de origen natural, origen natural
 synonyms:fi:luontainen, luonnollinen
 synonyms:it:naturale, di origine naturale, origine naturale
 
 synonyms:fr:élevé, élevée, élevés, élevées
 
+synonyms:en:from the end, after
 synonyms:fr:dès la fin, après
 
+synonyms:en:antibiotic, anti-biotic, anti-biotics, antibiotic treatment, use of antibiotics
 synonyms:fr:antibiotique, anti-biotique, anti-biotiques, anti-biotiques, traitement antibiotique, utilisation d'antibiotique
 
+synonyms:en:oviposition phase, oviposition period, oviposition
```

Comment:
We have no labels with "oviposition" so we should not add synonyms that won't be used.
https://world.openfoodfacts.org/labels?filter=oviposition -> 0 results